README.md

```diff
@@ -42,6 +42,8 @@ optional arguments:
                         /opt/License2Deploy/config.yml
   -s STACK_NAME, --stack STACK_NAME
                         Stack name if AutoScaling Group created via CloudFormation
+  -f FORCE, --force FORCE
```

Comment:
Hi Nick, we talked a little bit over here. Can we default this to false by default to preserve existing behavior and expose an option on the jenkins jobs to toggle this behavior on as needed. 
We'd also like to change "force" to something more descriptive / intuitive(We know it named as "force") in the ticket.